src/sparseml/pytorch/image_classification/export.py

```diff
@@ -149,6 +154,14 @@
     help="A path to a previous checkpoint to load the state from "
     "and resume the state for exporting",
 )
+@click.option(
+    "--labels_to_class-mapping",
+    "--labels-to-class_mapping",
+    type=str,
+    default=None,
+    help="Optional path to file containing the dataset-specific "
```

Comment:
nit: help string needs to be updated to match longer description string at top of file